src/index.html

```diff
@@ -1,12 +1,469 @@
+    <meta
+      name="viewport" content="width=device-width, initial-scale=1.0"
+    >
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a
+        href="http://localhost:8080/"
+        class="image">
+        <img
+          src="images/logo.png"
+          alt="moyo-logo"
+        >
+      </a>
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">apple</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#samsung"
+              class="nav__link">
+              samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#smartphones-phones"
+              class="nav__link">
+              смартфоны и телефоны
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#computers"
+              class="nav__link"
+              data-qa="nav-hover">
+              ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#planshets" class="nav__link">планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">фото</a>
+          </li>
+          <li class="nav__item">
+            <a href="#video" class="nav__link">видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main page__main">
+      <section class="main__wrapper">
+        <div class="product main__product" data-qa="card">
+          <div class="product__image-block">
+            <img
+              src="images/imac.jpeg"
+              alt="monitor-picture"
+              class="product__image"
+          >
+          </div>
+          <div class="product__titles">
+            <span class="product__item-title">
+              Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </span>
+            <span class="product__item-code">
+              Код товара: 195434
+            </span>
+          </div>
+
+          <div class="product__feedback">
+            <div class="product__stars">
+              <div class="product__star-active"></div>
+              <div class="product__star-active"></div>
+              <div class="product__star-active"></div>
+              <div class="product__star-active"></div>
+              <div class="product__star-not-active"></div>
+            </div>
```

Comment:
You should reuse your Stars block from previous task with modifier `stars--4` and corresponding styles.

Reply:
Every recommendation is done!